datus/storage/rdb/sqlite_backend.py

```
@@ -0,0 +1,324 @@
+                return cursor.rowcount
+        except sqlite3.IntegrityError as e:
+            raise IntegrityError(str(e)) from e
+
+    def _delete(self, table: str, where: Optional[WhereClause] = None) -> int:
+        where_sql, params = self._build_where(where)
+        sql = f"DELETE FROM {table}{where_sql}"
+        with self._auto_conn() as conn:
+            cursor = conn.execute(sql, params)
+            return cursor.rowcount
+
+    def _upsert(self, table: str, record: Any, conflict_columns: List[str]) -> None:
```

Ruff (0.15.2)

[warning] Unused method argument: `conflict_columns`

(ARG002)